tasks/variables/variables.js

```diff
@@ -1,40 +1,49 @@
 /**
  * TODO create variable named int and set value equal to 1
 */
-
+let int = 1;
 
 /**
  *  TODO create variable named int2 and set value equal int plus 1
  */
+let int2 = int + 1;
 
 /**
  * TODO create variable named str and set value equal to "John"
  */
+let str = "John";
 
 /**
  * TODO create variable named str2 and set value equal str plus " Dou"
  */
+let str2 = str + " Dou";
 
 /**
  * TODO create variable named bool and set value equal to comparison of int and int2 variables
  */
+let bool = int == int2;
 
 /**
  * TODO create variable named arr and set value array of numbers from 1 to 5
  */
+let arr = [1,2,3,4,5];
```

Comment:
it's better to use 'const' in the case of the arrays/objects

Reply:
Got it. Thanks.